manifests/config.pp

```diff
@@ -37,6 +37,17 @@
     mode   => '0750',
   }
 
+  if $keycloak::https {
```

Comment:
Could the `java_ks` type be used here to build the keystore?  Similar to here: https://github.com/treydock/puppet-module-keycloak/blob/master/manifests/truststore/host.pp#L20.